package/cluster.firewalld.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>SUSE YaST Cluster</short>
+  <description>This allows you to open various ports related to SUSE YaST Cluster module. Ports are opened for corosync, mgmtd, hawk, dlm and csync2.</description>
+  <port protocol="tcp" port="5403"/>
+  <port protocol="tcp" port="5560"/>
+  <port protocol="tcp" port="7630"/>
+  <port protocol="tcp" port="21064"/>
+  <port protocol="tcp" port="30865"/>
+</service>
```

Comment:
Please don't open the port 5403 for corosync-qdevice by default. It is only enabled with corosync-qdevice on and the port number also able to configure in UI.